examples/src/charmory_examples/object_detection/xview/test_xview_model.py

```diff
@@ -0,0 +1,203 @@
+import albumentations as A
+import numpy as np
+import torch
+from datasets import load_from_disk
+from datasets.filesystems import S3FileSystem
+import cv2
+
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+BATCH_SIZE = 1
+
+torch.set_float32_matmul_precision("high")
+
+LABEL_MAP_1 = {
+    0: 1,
+    1: 2,  
+    2: 3, 
+    3: 4,  
+    4: 5, 
+    5: 6, 
+
+}
+CLASSES_1 = [
+'None',
+'Passenger Vehicle',
+'Building',
+'Other',
+'Truck',
+'Engineering Vehicle',
+'Maritime Vessel',    
+]
+def load_huggingface_dataset():
+    s3 = S3FileSystem(anon=False)
+    train_dataset = load_from_disk("s3://armory-library-data/datasets/train/", fs=s3)
+
+    new_dataset = train_dataset.train_test_split(test_size=0.3, seed=1)
+    train_dataset, test_dataset = new_dataset["train"], new_dataset["test"]
+
+    return train_dataset, test_dataset
+train_dataset, test_dataset = load_huggingface_dataset()
+
+detection_threshold = 0.2
+
+from albumentations.pytorch import ToTensorV2
+def collate_fn(batch):
+    """
+    To handle the data loading as different images may have different number 
+    of objects and to handle varying size tensors as well.
+    """
+    return tuple(zip(*batch))
+def get_train_transform():
+    return A.Compose([
+
+        ToTensorV2(p=1.0),
+    ], bbox_params={
+        'format': 'pascal_voc',
+        'label_fields': ['labels']
+    })
+# define the validation transforms
+def get_valid_transform():
+    return A.Compose([
+        ToTensorV2(p=1.0),
+    ], bbox_params={
+        'format': 'pascal_voc', 
+        'label_fields': ['labels']
+    })
+
+
+
+
+class xview(torch.utils.data.Dataset):
```

Comment:
Is this class the same as the one in the other file?

Reply:
Yes thanks!